sys/conf/Makefile.mips

```diff
@@ -62,6 +64,8 @@ TRAMP_ELFSIZE=64
 .else
 TRAMP_ELFSIZE=32
 .endif
+# XXX-AM: Hack to clear trampoline flags
+TRAMP_EXTRA_LDFLAGS=${TRAMP_LDFLAGS:N-fuse-ld*}
```

Comment:
What's this for?

Reply:
It is an old hack that should have been reverted. Removed now.